http/serve_static.go

```diff
@@ -0,0 +1,57 @@
+package http
+
+import (
+	"html/template"
+	"net/http"
+)
+
+// StaticPages holds together all paths to a static pages
+type StaticPages struct {
+	Login          string
+	Registration   string
+	ForgotPassword string
+	ResetPassword  string
+}
+
+// ServeTemplate receives path to a template and serves it over http
+func (ar *apiRouter) ServeTemplate(path string) http.HandlerFunc {
+	tmpl, err := template.ParseFiles(path)
+	return func(w http.ResponseWriter, r *http.Request) {
+		if err != nil {
+			ar.Error(w, err, http.StatusInternalServerError, "")
+			return
+		}
+
+		err := tmpl.Execute(w, nil)
+		if err != nil {
+			ar.Error(w, err, http.StatusInternalServerError, "")
+		}
+	}
+}
+
+// ServeStaticPages serves static provided pages
+func ServeStaticPages(sp StaticPages) func(*apiRouter) error {
+	return func(ar *apiRouter) error {
+		return ar.serveStaticPages(sp)
+	}
+}
+
+// ServeDefaultStaticPages serves default HTML pages
+func ServeDefaultStaticPages() func(*apiRouter) error {
+	staticPages := StaticPages{
+		Login:          "../../static/login.html",
```

Comment:
I don't think it's a good default path. I recommend using subfolder in the working directory, like: `./static/login.html`

Reply:
We have 4 different working directories for each persistent layer, so I have to copy folder `static` into each working directory to make it work correctly.